tests/sys/fido2_ctap_hid/main.c

```diff
@@ -19,13 +19,13 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-#include "xtimer.h"
+#include "ztimer.h"
 
 #include "fido2/ctap/transport/ctap_transport.h"
 
 int main(void)
 {
     /* sleep in order to see early DEBUG outputs */
-    xtimer_sleep(3);
+    ztimer_sleep(ZTIMER_SEC, 3);
```

Comment:
you could use 
xtimer_compat.h

or 

ztimer_msec  

to not require an extra timer just for this startup wait

Reply:
done, changed it to zimer_sleep(ZTIMER_MSEC, 3000)